src/components/ui/tooltip/tooltip.scss

```diff
@@ -29,6 +38,20 @@ $triangle-size: 10px;
     top: $y-offset;
   }
 
+  &--no-delay {
+    transition-delay: 0s;
+  }
+}
+
+.pipeline-tooltip__text {
+  @extend %tooltip;
+
+  position: relative;
+  width: max-content;
+  max-width: calc(50vw - 150px);
+  padding: 12px 20px;
```

Comment:
Looking at the design, I think these values are off a little bit:

<img width="475" alt="image" src="https://github.com/kedro-org/kedro-viz/assets/16869061/f32962cb-4a76-4203-93fb-f924b7cc2021">

Reply:
Oh I see the warning now. It's there only for a moment and I didn't see it. 

I do agree sorting React props makes sense  but it doesn't for css props. It would be easier to read of sorted by display, layout, appearance and text props. 

Potentially we can add a script that auto-sorts the css props for us by given rules.